### ISSUE TITLE
Requiring 2FA?

### ISSUE THREAD

**Issue by IanLee1521** [LLNL/open-source-guidelines#9]
What do folks, e.g. @LLNL/admins think about requiring GitHub 2FA for the organization? I'm inclined to do this, but there are some potential issues:

- Some users don't have 2FA enabled, would need to enable or get booted from the org
  - Org members: https://github.com/orgs/LLNL/people?utf8=✓&query=%20two-factor%3Adisabled
  - Org collaborators: https://github.com/orgs/LLNL/outside-collaborators?utf8=✓&query=%20two-factor%3Adisabled
- What about locations that don't allow phones on site?
  - Response on this from GitHub support has been that users can use recovery codes as 1 time 2FA codes. Don't know of a better way...

**Comment by tgamblin** [comment]
I think it's a nonstarter as long as WCI folks won't be able to use their phones.

**Comment by davidbeckingsale** [comment]
Yeah, agree with @tgamblin on this... it's not going to work if people have to use their backup codes to log in.
